star-wars-characters/src/App.css

```diff
@@ -0,0 +1,40 @@
+.App {
```

Comment:
Character list was hard to see with this background.   Maybe use contrasting color